src/AWSSDK.MobileAnalytics.MobileAnalyticsManager.csproj

```diff
@@ -20,18 +20,9 @@
     <FileVersion>3.6.0.0</FileVersion>
   </PropertyGroup>
 
-  <Choose>
-    <When Condition=" '$(AWSKeyFile)' == '' ">
-      <PropertyGroup>
-        <AssemblyOriginatorKeyFile>../awssdk.dll.snk</AssemblyOriginatorKeyFile>
-      </PropertyGroup>
-    </When>
-    <Otherwise>
-      <PropertyGroup>
-        <AssemblyOriginatorKeyFile>$(AWSKeyFile)</AssemblyOriginatorKeyFile>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>../public.snk</AssemblyOriginatorKeyFile>
```

Comment:
why don't we have `<SignAssembly>true</SignAssembly>` here?

Reply:
It's already there on line 18.